packages/core/src/node/messaging/ipc-protocol.ts

```diff
@@ -45,13 +45,25 @@ export function checkParentAlive(): void {
 
 export const ipcEntryPoint = process.env[THEIA_ENTRY_POINT];
 
+const exclusion = [
```

Comment:
question: should it be there or only in plugin-ext ? (we call createIpcEnv and then we remove unwanted keys) ?

looks odd to have PLUGINS stuff inside code

Reply:
VS Code extensions need THEIA_PARENT_PID

Reply:
So, it is there 
```
childEnv[THEIA_PARENT_PID] = String(process.pid);
```